modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
```

Comment:
Why is this necessary?

Reply:
The child process is running as ocfbroker, so it will not automatically be killed on logout.